index.html

```diff
@@ -5,9 +5,21 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
- Kindly add information to your README file this requirement you will find it on your [Project Requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md) > [General Requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). If you need a template you can have it [here](https://github.com/microverseinc/readme-template).

Reply:
This issue is well resolved! ✔️